From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: remove wrong include path.

Don't use $(prefix)/include as include path. It can break things
when cross-compiling with DESTDIR and prefix=/usr

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

# 20091215 mol: this patch is already upstream: 86a7baa4156dc9569b7dde51b3042b7fd8093821

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-dummy-0.3.2/configure.ac
===================================================================
--- xf86-video-dummy-0.3.2.orig/configure.ac
+++ xf86-video-dummy-0.3.2/configure.ac
@@ -58,7 +58,7 @@ PKG_CHECK_MODULES(XORG, [xorg-server >= 
 sdkdir=$(pkg-config --variable=sdkdir xorg-server)
 
 CFLAGS="$CFLAGS $XORG_CFLAGS "' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
